floris/parallel_floris_model_2.py

```diff
@@ -0,0 +1,309 @@
+from __future__ import annotations
+
+import copy
+import warnings
+from pathlib import Path
+from time import perf_counter as timerpc
+
+import numpy as np
+import pandas as pd
+
+from floris.core import State
+from floris.floris_model import FlorisModel
+from floris.optimization.yaw_optimization.yaw_optimizer_sr import YawOptimizationSR
+
+
+class ParallelFlorisModel(FlorisModel):
+    """
+    This class mimics the FlorisModel, but enables parallelization of the main
+    computational effort.
+    """
+
+    def __init__(
+        self,
+        configuration: dict | str | Path | FlorisModel,
+        interface: str | None = "multiprocessing",
+        max_workers: int = -1,
+        n_wind_condition_splits: int = 1,
```

Comment:
Should this be -1?

Reply:
Good catch! That was a typo and a bug---this default should be -1. I'm just rerunning tests etc with that change.